PowerEditor/installer/nativeLang/bulgarian.xml

```diff
@@ -997,6 +997,7 @@
 					<Item id="6903" name="Диалога за търсене остава отворен, след като резултатите бъдат изведени в прозореца"/>
 					<Item id="6904" name="Потвърждение при &quot;Замяна на всичко във всички отворени&quot; документи"/>
 					<Item id="6905" name="Без преместване към следващото повторение при замяна"/>
+					<Item id="6906" name="Резултати от търсене: показване само на един запис на намерен ред (не се прилага за режим RTL)"/>
```

Comment:
"(not applied to RTL mode)" has been removed in this entry, see: https://github.com/notepad-plus-plus/notepad-plus-plus/commit/059bca3315dbd71fbf235c8bc87eaacbd87a3cec
Sorry for the inconvenience.